alphabase/psm_reader/dia_psm_reader.py

```diff
@@ -19,32 +17,8 @@ class SpectronautReader(MaxQuantReader):
 
     _reader_type = "spectronaut"
     _add_unimod_to_mod_mapping = True
-
-    def __init__(  # noqa: PLR0913 many arguments in function definition
-        self,
-        *,
-        column_mapping: Optional[dict] = None,
-        modification_mapping: Optional[dict] = None,
-        fdr: float = 0.01,
-        keep_decoy: bool = False,
-        fixed_C57: bool = False,  # noqa: N803 TODO: make this  *,fixed_c57  (breaking)
-        mod_seq_columns: Optional[List[str]] = None,
-        rt_unit: str = "minute",
-        **kwargs,
-    ):
-        """Initialize SpectronautReader."""
-        super().__init__(
-            column_mapping=column_mapping,
-            modification_mapping=modification_mapping,
-            fdr=fdr,
-            keep_decoy=keep_decoy,
-            mod_seq_columns=mod_seq_columns,
-            fixed_C57=fixed_C57,
-            rt_unit=rt_unit,
-            **kwargs,
-        )
-
-        self._min_max_rt_norm = True
+    _min_max_rt_norm = True
```

Comment:
I think this is something we might rather want to do on an alphabase level as post processing